### ISSUE TITLE
invalid parameter: allow_virtual

### ISSUE THREAD

**Issue by peterbeck** [saz/puppet-ssh#104]
Hi there,

I got following errors when applying the module on a Debian Wheezy machine:
```
Error 400 on SERVER: Invalid parameter allow_virtual at 
/etc/puppet/modules/ssh/manifests/server/install.pp:8
```
```
Error 400 on SERVER: Invalid parameter allow_virtual at 
/etc/puppet/modules/ssh/manifests/client/install.pp:7
```
When I comment both parameters in install.pp it seems to work fine. I can't find these parameters anywhere else in the module. What am I doing wrong here?

**Comment by wazoo** [comment]
I had this issue, I think it is because that value didn't exist on older versions of puppet (I had agent version 3.4.x and upgrading to 3.7.x resolved this)
https://docs.puppetlabs.com/references/3.4.latest/type.html#package
vs
https://docs.puppetlabs.com/references/3.7.latest/type.html#package

Looks like it first shows up in 3.6.latest.

**Comment by sbadia** [comment]
Hi here, this issue can be closed (fixed by #98), thanks!
